WordPress/Classes/ViewRelated/NUX/NoSitesView.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+
+@objc protocol NoSitesViewDelegate {
+    func addSiteButtonPressed()
+}
+
+@objc class NoSitesView: UIView {
+
+    // MARK: - Properties
+
+    @objc weak var delegate: NoSitesViewDelegate?
+
+    @IBOutlet weak var noSitesTitle: UILabel!
+    @IBOutlet weak var addSiteButton: LoginButton!
+    @IBOutlet weak var titleLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet weak var titleTrailingConstraint: NSLayoutConstraint!
+
+    // MARK: - Init
+
+    @objc class func instanceFromNib() -> NoSitesView {
+        let view = UINib(nibName: "NoSitesView", bundle: nil).instantiate(withOwner: nil, options: nil).first as! NoSitesView
```

Comment:
The `as!` hurts me 😜 I have a hangup about `!`, fwiw, but this reads as `if loading the nib fails, crash`. If we were to return an unconfigured `NoSitesView` instead it would be ugly but wouldn't crash 😉

Reply:
Sorry! That was an oversight - I don't _intentionally_ give you a stroke. 😉